test/controllers/users_controller_test.rb

```diff
@@ -338,7 +338,7 @@ class UsersControllerTest < ActionController::TestCase
 
         get :edit, params: { id: user }
 
-        assert_select "a[href='#{edit_user_two_step_verification_mandation_path(user)}']", text: "Mandate 2-step verification for this user"
+        assert_select "a[href='#{edit_user_two_step_verification_mandation_path(user)}']", text: "Turn on 2-step verification for this user"
```

Comment:
I half wonder if this route name should change to remove the word "mandate". But I don't think `edit_user_two_step_verification_turn_on_path` reads particularly well, so maybe we can leave it.

Reply:
As before, I think "mandate" is a more accurate description of what this functionality does and I don't mind that it doesn't match the copy exactly.